src/lib/sections/Navigation/CollapseToggle/CollapseToggle.test.tsx

```diff
@@ -5,24 +5,43 @@ import { vi } from "vitest";
 import { Navigation } from "@/lib/sections/Navigation/Navigation";
 
 it("renders without crashing", () => {
-  render(<Navigation.CollapseToggle isCollapsed={false} setIsCollapsed={vi.fn()} />)
-  expect(screen.getByRole("button", { name: "collapse main navigation" })).toBeInTheDocument();
-})
+  render(
+    <Navigation.CollapseToggle isCollapsed={false} setIsCollapsed={vi.fn()} />,
+  );
+  expect(
+    screen.getByRole("button", { name: "collapse main navigation" }),
+  ).toBeInTheDocument();
+});
 
 it("changes the aria label when collapsed", () => {
-  render(<Navigation.CollapseToggle isCollapsed={true} setIsCollapsed={vi.fn()} />)
-  expect(screen.getByRole("button", { name: "expand main navigation" })).toBeInTheDocument();
-})
+  render(
+    <Navigation.CollapseToggle isCollapsed={true} setIsCollapsed={vi.fn()} />,
+  );
+  expect(
+    screen.getByRole("button", { name: "expand main navigation" }),
+  ).toBeInTheDocument();
+});
 
 it("displays a tooltip on hover", async () => {
-  render(<Navigation.CollapseToggle isCollapsed={false} setIsCollapsed={vi.fn()} />)
-  await userEvent.hover(screen.getByRole("button", { name: "collapse main navigation" }))
-  expect(screen.getByRole("tooltip", { name: /collapse/i })).toBeInTheDocument();
-})
+  render(
+    <Navigation.CollapseToggle isCollapsed={false} setIsCollapsed={vi.fn()} />,
+  );
+  await userEvent.hover(
+    screen.getByRole("button", { name: "collapse main navigation" }),
+  );
+  expect(screen.getByRole("button", { name: /collapse/i })).toBeInTheDocument();
```

Comment:
This isn't testing for tooltip presence, as the test name suggests

Reply:
Oops 🙈 Good catch, thanks!